test-cases/scale/longevity-scale-5dcs-cluster-xl.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 6000
+
+prepare_write_cmd:  ["cassandra-stress write cl=LOCAL_QUORUM n=20971520 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=20 -pop seq=1..20971520 -col 'n=FIXED(5) size=FIXED(64)' -log interval=5",
```

Comment:
Any reason we use SizeTieredCompactionStrategy? In the future it'll be the same as ICS anyway.

Reply:
only reason that it was default for most of sct tests. @mykaul , you suggest to change to ICS?

Reply:
Changed to IncrementalCompactionStrategy